virtwho/virt/esx/esx.py

```diff
@@ -40,6 +40,11 @@
 from virtwho import virt
 from virtwho.config import VirtConfigSection
 
+try:
+    from urllib import unquote as urldecode
```

Comment:
I guess that this import is intended for Python2.7. Do we plan to still support Python2.7?
Anyway, shouldn't we try to import module for Python3 and if it is not possible, then try to import module for Python2.7?

Reply:
We will likely backport this to 7.9 at some point so I think it is worth leaving. I did switch the order of the imports.

Reply:
Oh, in this case it make sense.